[CI] push-android-runtime-build failed!

Last 50 lines of output:
```[...truncated 97.42 KB...]
  ^
/externals/gtest/googletest/include/gtest/gtest_pred_impl.h:147:28: note: expanded from macro 'GTEST_PRED_FORMAT2_'
  GTEST_ASSERT_(pred_format(#v1, #v2, v1, v2), \
                           ^
3 errors generated.
tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/build.make:62: recipe for target 'tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/src/FourOneOpModelSetInput.cc.o' failed
make[3]: *** [tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/src/FourOneOpModelSetInput.cc.o] Error 1
make[3]: *** Waiting for unfinished jobs....
6 errors generated.
tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/build.make:110: recipe for target 'tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/src/load_model.cc.o' failed
make[3]: *** [tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/src/load_model.cc.o] Error 1
[ 95%] Building CXX object runtime/libs/tflite/CMakeFiles/nnfw_lib_tflite.dir/src/FeatureView.cpp.o
[ 95%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/kernel/StridedSliceLayer.cc.o
[ 95%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/kernel/SubLayer.cc.o
[ 95%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/kernel/TanhLayer.cc.o
[ 95%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/kernel/TransposeLayer.cc.o
[ 96%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/kernel/UnpackLayer.cc.o
[ 96%] Building CXX object runtime/onert/backend/cpu/CMakeFiles/onert_backend_cpu.dir/operand/Tensor.cc.o
[ 96%] Building CXX object runtime/libs/tflite/CMakeFiles/nnfw_lib_tflite.dir/src/Quantization.cpp.o
[ 97%] Building CXX object runtime/libs/tflite/CMakeFiles/nnfw_lib_tflite.dir/src/TensorShapeUtils.cpp.o
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
make[2]: *** [tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:2800: recipe for target 'tests/nnfw_api/CMakeFiles/nnfw_api_gtest.dir/all' failed
[ 97%] Building CXX object runtime/libs/tflite/CMakeFiles/nnfw_lib_tflite.dir/src/interp/FunctionBuilder.cpp.o
[ 97%] Building CXX object runtime/libs/tflite/CMakeFiles/nnfw_lib_tflite.dir/src/interp/FlatBufferBuilder.cpp.o
[ 97%] Linking CXX shared library libbackend_acl_cl.so
[ 97%] Linking CXX shared library libneuralnetworks.so
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
[ 97%] Built target onert
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
[ 97%] Built target onert_backend_acl_cl
[ 97%] Linking CXX static library libnnfw_lib_tflite.a
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
[ 97%] Built target nnfw_lib_tflite
[ 97%] Linking CXX shared library libbackend_acl_neon.so
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
[ 97%] Built target onert_backend_acl_neon
[ 97%] Linking CXX shared library libbackend_cpu.so
make[3]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
[ 97%] Built target onert_backend_cpu
make[2]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
Makefile:140: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/home/jenkins_slave/workspace/nnfw/master/push-android-runtime-build/Product/aarch64-android.debug/obj'
make: *** [build_internal] Error 2
Makefile.template:135: recipe for target 'build_internal' failed
Process leaked file descriptors. See https://jenkins.io/redirect/troubleshooting/process-leaked-file-descriptors for more information
Build step 'Execute shell' marked build as failure
```

[View full output](http://npuci.mooo.com:8080/job/nnfw/job/master/job/push-android-runtime-build/35/)

--- a/tests/nnfw_api/src/fixtures.h
+++ b/tests/nnfw_api/src/fixtures.h
@@ -89,7 +89,7 @@ protected:
   {
     ValidationTest::SetUp();
 
-    auto model_path = ModelPath::get().getModelAbsolutePath(MODEL_ONE_OP_IN_TFLITE);
+    auto model_path = ModelPath::get().getModelAbsolutePath(MODEL_ADD);
     for (auto &obj : _objects)
     {
       ASSERT_EQ(nnfw_create_session(&obj.session), NNFW_STATUS_NO_ERROR);

--- a/tests/nnfw_api/src/load_model.cc
+++ b/tests/nnfw_api/src/load_model.cc
@@ -34,9 +34,8 @@ TEST_F(ValidationTestSessionCreated, neg_load_session_001)
 
 TEST_F(ValidationTestSessionCreated, neg_load_session_002)
 {
-  ASSERT_EQ(nnfw_load_model_from_file(
-                nullptr, // session is null
-                ModelPath::get().getModelAbsolutePath(MODEL_ONE_OP_IN_TFLITE).c_str()),
+  ASSERT_EQ(nnfw_load_model_from_file(nullptr, // session is null
+                                      ModelPath::get().getModelAbsolutePath(MODEL_ADD).c_str()),
             NNFW_STATUS_ERROR);
 }
 

